core/src/main/resources/lib/hudson/progressBar.jelly

```diff
@@ -32,19 +32,20 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:x="jelly:xml" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <table class="progress-bar ${attrs.red?'red':null}" id="${attrs.id}" tooltip="${attrs.tooltip}" href="${attrs.href}" style="${attrs.href!=null ? 'cursor:pointer' : null}">
+  <div class="app-progress-bar ${attrs.large?'app-progress-bar-large':null} ${attrs.red?'red':null}"
```

Comment:
Would be nice if the tag for the progress bar changed if it's a link - see https://github.com/janfaracik/jenkins/blob/8e88712bf2f7de338773ed03723c6682e5ef022a/core/src/main/resources/lib/hudson/progressBar.jelly#L44

Reply:
done